api/room/schema.py

```diff
@@ -45,6 +46,7 @@ class Arguments:
         capacity = graphene.Int()
         image_url = graphene.String()
         calendar_id = graphene.String()
+        location = graphene.String()
```

Comment:
Any reason why the location field was added, thought this was to be found by following back the foreign key relationship?